.github/workflows/publish-idf-rust-tags.yml

```
@@ -71,35 +20,35 @@ jobs:
         board: ["esp32", "esp32s2", "esp32s3", "esp32c3", "all"]
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Build Docker image
         if: matrix.board != 'all'
         run: |
           docker image build --tag idf-rust:${{ matrix.board }}_${{ github.event.inputs.toolchain-version }} \
-           --file idf-rust.Containerfile --build-arg XTENSA_TOOLCHAIN_VERSION=${{ github.event.inputs.toolchain-version }} \
-           --build-arg ESP_BOARD=${{ matrix.board }} .
-      - name: Build template project using generated tag (master)
+           --file idf-rust.Containerfile --build-arg ESP_BOARD=${{ matrix.board }} --build-arg XTENSA_VERSION=${{ github.event.inputs.toolchain-version }} --build-arg GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} .
+      - name: Build template projects using generated tag
         if: matrix.board != 'all'
         run: |
           docker run --user esp --mount type=bind,source="$(pwd)/build-template-project.sh",target=/workspace/build-template-project.sh,consistency=cached \
-                 --rm idf-rust:${{ matrix.board }}_${{ github.event.inputs.toolchain-version }} /bin/bash /workspace/build-template-project.sh esp-template ${{ matrix.board }}
+                 --rm idf-rust:${{ matrix.board }}_${{ github.event.inputs.toolchain-version }} /bin/bash /workspace/build-template-project.sh ${{ matrix.board }}
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v2
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
       - name: Login to DockerHub
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_TOKEN }}
       - name: Build - ${{ matrix.board }}_${{ github.event.inputs.toolchain-version }} tag
-        uses: docker/build-push-action@v3
+        uses: docker/build-push-action@v4
         with:
           context: .
           file: idf-rust.Containerfile
           build-args: |
-            XTENSA_TOOLCHAIN_VERSION=${{ github.event.inputs.toolchain-version }}
             ESP_BOARD=${{ matrix.board }}
+            GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
+            XTENSA_VERSION=${{ github.event.inputs.toolchain-version }}
           tags: espressif/idf-rust:${{ matrix.board }}_${{ github.event.inputs.toolchain-version }},espressif/idf-rust:${{ matrix.board }}_latest
-          push: true
+          push: false
```

We forgot to change `push` value back to `true`:
```suggestion
          push: true
```